network_server:
  object_detect_service_name: object_detect_service
  network_cloud_service_name: network_cloud_service
  network_semantic_service_name: network_semantic_service
  yolov3:
    config_path: /home/ericlab/tsuchida/ros_package/study_ws/src/denso_estimation/network/src/network/object_detection/yolov3/config/yolov3_denso.yaml
  ssd:
    config_path: /home/ericlab/tsuchida/ros_package/study_ws/src/denso_estimation/network/src/network/object_detection/ssd/config/yaml/vgg_ssd512_voc0712.yaml
    threshold: 0.95